gordo_client/io.py

```diff
@@ -81,7 +89,7 @@ def _handle_response(resp: requests.Response, resource_name: Optional[str] = Non
         if _is_json_response(resp):
             return resp.json()
         elif _is_azure_login_response(resp):
-            raise BadGordoRequest(f"Azure login page response found while fetching resource: {resource_name}.")
+            raise BadGordoResponse(f"Azure login page response found while fetching resource: {resource_name}.")
```

Comment:
`gordo-client` is a platform agnostic open-source client, so there is should not be any mentions about azure
I suggest removing this elsif branch.